spec/server/functional/prepaid.spec.coffee

```diff
@@ -176,6 +227,66 @@ describe 'POST /db/prepaid/:handle/redeemers', ->
     expect(student.get('coursePrepaid')?.type).toEqual('starter_license')
     done()
 
+  describe '.canReplace', ->
+    beforeEach utils.wrap (done) ->
+      yield utils.clearModels([Prepaid])
+      yield utils.loginUser(@admin)
+      @starter = yield utils.makePrepaid({
```

Comment:
This is a unit test. Should go in /spec/server/unit/Prepaid.spec.coffee (and be created if needed).